src/helpers/figures/charts/chart_common_line_scatter.ts

```diff
@@ -150,20 +155,12 @@ function getLineOrScatterConfiguration(
       title: getChartAxisTitleRuntime(chart.axesDesign?.x),
     },
   };
-  const formatCallback = (value) => {
```

Comment:
There's still this piece of code duplicated in `combo_chart.ts`. It could be replaced with the new `formatTickValue`. in this PR or in master?

Reply:
done here :)